Remotion/Web/Development.WebTesting/DownloadInfrastructure/Default/DefaultDownloadHelper.cs

```diff
@@ -186,12 +186,10 @@ private void CleanUpUnmatchedDownloadedFiles ([NotNull] IEnumerable<string> unma
     private void WaitUntilTrueOrTimeout (Func<bool> func, TimeSpan timeout, TimeSpan interval)
     {
       var success = false;
-      var elapsed = 0d;
-      while (!success && elapsed < timeout.TotalMilliseconds)
+      for (var elapsed = 0d; !success && elapsed < timeout.TotalMilliseconds; elapsed += interval.TotalMilliseconds)
       {
         success = func.Invoke();

```

Comment:
```
if (func.Invoke())
  return;
```

Has a little too much going on in one line, wouldn't you agree?

Reply:
am confused, this simple IF here?

Reply:
Since you think it's fine, I've gone ahead and inlined it 👍

Reply:
okay, there were two unrelated conversations for the same line :/

I can live with both. the if (operation()) return xx; pattern is typical for IO stuff, so i'm used to it. It might be that it's actually a style violation but honestly, it's an edgecase.